docs/source/api.rst

```diff
@@ -3,6 +3,7 @@ stdatamodels API
 
 .. automodapi:: stdatamodels
 .. automodapi:: stdatamodels.exceptions
+.. automodapi:: stdatamodels.dqflags
```

Comment:
Minor comment but this adds the docstring of `dqflags` to the docs. That docstring (not modified in this PR so I can't add a suggestion there) is incorrect:

> The data structure that stores bit flags is just the standard Python `int`,
which provides 32 bits.

To be pedantic I'd argue the array of values are "flags" and each element is a [bit field](https://en.wikipedia.org/wiki/Bit_field). Perhaps we modify the docstring to:

```
Interpret JWST data quality flags.

The flags are binary-packed structures representing information about a given element (a bit field). This information can be mapped to and from plain text mnemonics using `astropy.nddata.bitmask.interpret_bit_flags` and `stdatamodels.dqflags.dqflags_to_mnemonics`.
```